src/Stryker.Core/Stryker.Core/Compiling/RollbackProcess.cs

```diff
@@ -167,7 +167,7 @@ private SyntaxTree RemoveMutantIfStatements(SyntaxTree originalTree, ICollection
                 if (mutationIf == null)
                 {
                     var errorLocation = diagnostic.Location.GetMappedLineSpan();
-                    _logger.LogWarning("Stryker.NET encountered an compile error in {0} with message: {1} (Source code: {2})", errorLocation.Path, diagnostic.GetMessage(), brokenMutation);
+                    _logger.LogWarning($"Stryker.NET encountered an compile error in {errorLocation.Path} (at {errorLocation.StartLinePosition.Line}:{errorLocation.StartLinePosition.Character}) with message: {diagnostic.GetMessage()} (Source code: {brokenMutation})");
```

Comment:
I added more location information in case of build error so we can provide diagnostic faster.